lib/redlock/client.rb

```diff
@@ -228,6 +258,22 @@ def lock_instances(resource, ttl, options)
       end
     end
 
+    def try_get_remaining_ttl(resource, value)
+      ttls, time_elapsed = timed do
+        @servers.map { |s| s.get_remaining_ttl resource, value }.compact
```

Comment:
something like

```ruby
@servers.lazy.map { |s| s.get_remaining_ttl resource, value }.select(&:itself).take(@quorum).to_a
```

should work, right?

Reply:
Yeah, this looks good 👍